src/components/ChangePlan/CheckoutProcess.js

```diff
@@ -0,0 +1,361 @@
+                      label={`${_('checkoutProcessForm.contact_information_industry')}`}
+                      fieldName={fieldNames.industry}
+                      id="industry"
+                      className="field-item--50"
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+            </FieldGroup>
+            {formSubmitted ? (
+              <div className="dp-wrap-confirmation" style={containerMessageSucces}>
+                <div className="dp-msj-confirmation bounceIn" style={flextStartSyle}>
+                  <p>{_('checkoutProcessForm.success_msg')}</p>
+                </div>
+              </div>
+            ) : null}
+            <div className="dp-footer-form" style={flextStartSyle}>
+              <div className="dp-action">
+                <SubmitButton className="dp-button button-medium primary-green">
+                  {_('checkoutProcessForm.save_continue')}
+                </SubmitButton>
+              </div>
+            </div>
+          </fieldset>
+        </FormWithCaptcha>
+      ) : null}
+    </div>
+  );
+};
+
+const CheckoutProcess = ({
+  location,
+  dependencies: { planService, dopplerUserApiClient, appSessionRef },
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const { pathType, planType } = useRouteMatch().params;
+
+  const [steps, setSteps] = useState({
+    contactInformation: true,
+    billingInformation: false,
+    paymentInformation: false,
+  });
+
+  const submitContactInformationForm = async (values) => {
+    const result = await dopplerUserApiClient.createOrUpdateContactInformation(values);
+
+    if (result.success) {
+      setActiveStep();
+    }
+
+    return result;
+  };
+
+  const setActiveStep = (target) => {
+    switch (target) {
+      case 'contact-information':
+        setSteps({
+          contactInformation: true,
+          billingInformation: false,
+          paymentInformation: false,
+        });
+        break;
+      case 'billing-information':
+        setSteps({
+          contactInformation: false,
+          billingInformation: true,
+          paymentInformation: false,
+        });
+        break;
+      case 'payment-information':
+        setSteps({
+          contactInformation: false,
+          billingInformation: false,
+          paymentInformation: true,
+        });
+        break;
+    }
+  };
+
+  const [state, setState] = useState({ loading: true });
+
+  useEffect(() => {
+    const mapCountry = (country) => {
+      return {
+        key: country.id,
+        value: country.name,
+      };
+    };
+
+    const fetchData = async () => {
+      const contactInformationResult = await dopplerUserApiClient.getContactInformationData();
+      const countriesResult = await dopplerUserApiClient.getCountriesData();
+
+      const countries = countriesResult.success ? countriesResult.value.map(mapCountry) : [];
```

Comment:
How about doing the mapping here directly?
```suggestion
      const countries = countriesResult.success ? countriesResult.value.map((country)=>{ key:country.id, value:country.name }) : [];
```

Reply:
Creo que no va a ser necesario ese código. Deberíamos usar los países que ya conoce la WebApp, sin ir al backend para obtenerlos.

Reply:
You could also do this:
```suggestion
      const defaultValue = { key: '', value: _('checkoutProcessForm.empty_option_select') };
      const countries = countriesResult.success ? [ defaultValue, ...countriesResult.value.map((country)=>{ key:country.id, value:country.name }) : [];
```

Reply:
@andresmoschini quizás se pudiese usar un contexto para almacenar los paises e implementar un componente tipo **`CountryField`** que se conecte a ese contexto, de tal forma que si se va a usar el campo `country` en algún formulario, ya toda esa lógica esté implementada. Como una mejora quizás, ya es algo que se pudiese atacar en otro task.

Reply:
No se si es necesario el contexto, tranquilamente podría ser un módulo (una capa sobre lo que hay ahora).

Estoy de acuerdo en que se podría hacer en otro PR.

Pero no estoy de acuerdo en hacer un GET de countries. Si quiere hacer un trabajo más pequeño, que sea un input de solo texto donde se introduzca el código de país.

Reply:
Genial! Mucho mejor! :+1: